### ISSUE TITLE
Unwanted Helvetica font

### ISSUE THREAD

**Issue by maguayo** [xhtml2pdf/xhtml2pdf#200]
No matter what font I use, there is always Helvetica and it's not embed, so most of printing companies can not print the document if a font is missing.

**Comment by teeberg** [comment]
You can specify another font like this:
```html
<style type="text/css">
    @font-face {
        font-family: "Arial";
        src: url('/absolute/path/to/your/fonts/Arial.ttf');
    }
    body {
        font-family: Arial, sans-serif;
    }
</style>
```

**Comment by benjaoming** [comment]
Solution given by @teeberg is correct. Either use a font native to the system or tell xhtml2pdf about a ttf font.

**Comment by MarcosAguayo** [comment]
Not solved. Try it yourself you'll see that the Helvetica will apear. It's imposible to delete this font

**Comment by benjaoming** [comment]
I think you're right, sorry I was assuming something too fast. It seems font substitution doesn't work.

**Comment by maguayo** [comment]
@jsonn Not all the printers does. I worked with Ingram (printing as demand) company. And they don't accept it. Also With other 2 companies I worked they require to append the font, or remove it.

**Comment by luisza** [comment]
This a long time issue opened, I agree with the solution so this will be closed.

**Comment by maguayo** [comment]
That didn't solve it, FYI, I had to create a ghostscript to remove the unused font.

**Comment by luisza** [comment]
Hi, take a look [here](https://github.com/xhtml2pdf/xhtml2pdf/blob/6a4b31979fe9ecf75f5776568f9ee032935966d6/xhtml2pdf/default.py#L389) ,
you can set your own default CSS for you file.
Pisa has a parameter called default_css that help you to provide a default css file, so you can remove Helvetica from html tag.

**Comment by luisza** [comment]
Also, I am coding with my team some font features, so if you have a example of your problem, we can check it and try to fix it on this days.

**Comment by maguayo** [comment]
@luisza Don't worry I had the problem 6 years ago, I fix it removing the unused font with a ghostscript, but the problem persists. This is the stackoverflow link with a more detailed problem: https://stackoverflow.com/questions/25203219/xhtml2pdf-doesnt-embed-helvetica

**Comment by luisza** [comment]
I guest this issue was fixed on 0.2.5 release :), sorry the delay.
